lib/worker.js

```diff
@@ -25,11 +25,14 @@ const { Sampler } = require('@adobe/metrics-sampler');
 const { metrics: cgroupMetrics, cpu} = require('@adobe/cgroup-metrics');
 
 const CLEANUP_FAILED_EXIT_CODE = 231;
+const TIMEOUT_EXIT_CODE = 124;
```

Comment:
Any reason for this number in particular?

It seems we have very random numbers here, 231, 124... maybe keep them more systematic?

Reply:
The reason I chose exit code 124 is because this is the exit code the [`timeout`](https://www.howtogeek.com/423286/how-to-use-the-timeout-command-on-linux/#:~:text=The%20exit%20code%20is%20124,code%20from%20timeout%20is%20zero.) function on Linux uses if it times out. It was slightly related to timing out so that is why I decided to use it. I am fine with changing to a different number. What would you suggest?